Makefile.am

```diff
@@ -5565,14 +5565,12 @@ dist_dbuspolicy_DATA += \
 dist_dbussystemservice_DATA += \
 	src/resolve/org.freedesktop.resolve1.service
 
-SYSTEM_UNIT_ALIASES += \
-	systemd-resolved.service dbus-org.freedesktop.resolve1.service
-
 BUSNAMES_TARGET_WANTS += \
 	org.freedesktop.resolve1.busname
 
 GENERAL_ALIASES += \
 	$(systemunitdir)/systemd-resolved.service $(pkgsysconfdir)/system/multi-user.target.wants/systemd-resolved.service
+	$(systemunitdir)/systemd-resolved.service $(pkgsysconfdir)/system/dbus-org.freedesktop.resolve1.service
```

Comment:
This one head my scratching my head for a moment. The description of the commit seems to bear little relation to the change. Also, there's a backslash missing in this chunk, so this patch doesn't work. I'll repush with this fixed and a new commit message.

I think we might need to do the same for other dbus-activated services, but that can be done separately.

Looks good otherwise.